ci-cd/build-nodejs-project/action.yml

```diff
@@ -32,6 +35,15 @@ runs:
     # npm install, build
     - shell: bash
       run: |
+        BUILD_ENVS=$(cat <<'EOF'
+        ${{ inputs.dsb-build-envs }}
+        EOF
+        )
+        
+        # Helper functions
+        # Get field value from BUILD_ENVS safely
+        function get-val { echo "${BUILD_ENVS}" | jq -r --arg name "$1" '.[$name]'; }
+        
```

Comment:
Slettes når vi kan lene oss på at `npm-test-e2e` alltid er definert og GH kan hente den for oss direkte.
```suggestion
```